core/client/fs/src/main/java/alluxio/client/block/stream/DataReader.java

```diff
@@ -28,6 +28,13 @@
    */
   DataBuffer readChunk() throws IOException;
 
+  /**
+   * Try to read a chunk without waiting for data ready. 
+   *
+   * @return the data buffer or null if EOF is reached or data is not ready.
+   */
+  DataBuffer readChunkNoWait() throws IOException;
```

Comment:
how about naming this method : `readChunkIfReady`

Reply:
Renamed